src/convertToObject.js

```diff
@@ -6,7 +6,15 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString
+    .split(';')
+    .map((pair) => pair.split(':').map((str) => str.trim()))
```

Comment:
Ensure that the `map` method's callback function handles cases where there might be more than one colon in the style declaration. This could occur if a style's value contains a colon, such as in a `content` property.